### PR TITLE
add python package bert-tensorflow

### PR DESCRIPTION
**Pull request by karimmohraz**
Checklist
- [X ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [ X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [ X] Source is from official source
- [ X] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [ X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ X] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by karimmohraz** [comment]
you can list me as a maintainer

File: recipes/bert-tensorflow/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
+{% set name = "bert-tensorflow" %}
+{% set version = "1.0.1" %}
+{% set build_num = 1 %}
```
Comment (scopatz) [inline recipes/bert-tensorflow/meta.yaml:3]: This should be zero, and doesn't need to be a variable.

File: recipes/bert-tensorflow/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 979ab38715be88bc95483654994c8bbb85acdbfdc60ca1a0ff90cfe8c8f95ea8
+
+build:
+  noarch: python
+  number: {{ build_num }}
+  script: "{{ PYTHON }} -m pip install . -vv"
+  string: py{{ py }}_h{{ PKG_HASH }}_{{ build_num }}
```
Comment (scopatz) [inline recipes/bert-tensorflow/meta.yaml:17]: This line should be removed

File: recipes/bert-tensorflow/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  imports:
+    - bert
+
+about:
+  home: "https://github.com/google-research/bert"
+  license: Apache 2.0
+  license_family: APACHE
+  license_file: LICENSE
+  summary: "BERT"
+  doc_url:
```
Comment (scopatz) [inline recipes/bert-tensorflow/meta.yaml:38]: Please remove blank entries

File: recipes/bert-tensorflow/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+test:
+  imports:
+    - bert
+
+about:
+  home: "https://github.com/google-research/bert"
+  license: Apache 2.0
+  license_family: APACHE
+  license_file: LICENSE
+  summary: "BERT"
```
Comment (scopatz) [inline recipes/bert-tensorflow/meta.yaml:35]: Please have a more meaningful summary

**Comment by karimmohraz** [comment]
@conda-forge-admin, please ping team
Performed the requested changes.

File: recipes/bert-tensorflow/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+    - six
+
+test:
+  imports:
+    - bert
+
+about:
+  home: "https://github.com/google-research/bert"
+  license: Apache 2.0
+  license_family: APACHE
```
Comment (chrisburr) [inline recipes/bert-tensorflow/meta.yaml:33]: The SPDX identifier has a dash, you can then remove the `license_family` field:
```suggestion
  license: Apache-2.0
```

File: recipes/bert-tensorflow/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python
+    - six
```
Comment (chrisburr) [inline recipes/bert-tensorflow/meta.yaml:21]: This probably isn't needed in `host`:
```suggestion
```

File: recipes/bert-tensorflow/meta.yaml [hunk 7]
```diff
@@ -18,10 +18,8 @@ requirements:
   host:
     - pip
     - python
-    - six
   run:
     - python
-    - six
```
Comment (chrisburr) [inline recipes/bert-tensorflow/meta.yaml]: You still need it in `run`

File: recipes/bert-tensorflow/meta.yaml [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
```
Comment (chrisburr) [inline recipes/bert-tensorflow/meta.yaml:22]: You still need it in `run`:
```suggestion
    - python
    - six
```

**Pull request merged**